packages/react-app-revamp/hooks/useContest/v3v4/contracts.ts

```diff
@@ -17,10 +18,10 @@ export function getContracts(contractConfig: any, version: number) {
 
   let contractFunctionNames = [...commonFunctionNames];
 
-  if (version >= 4) {
+  if (compareVersions(version.toString(), "4.0") >= 0) {
     contractFunctionNames = [...contractFunctionNames, ...v4FunctionNames];
   }
-  if (version >= 4.2) {
```

Comment:
This is another case of this issue which results in the wrong function names being set on any contracts between 4.10 and 4.19 inclusive.